### ISSUE TITLE
review usage of explicit `H1` in the components

### ISSUE THREAD

**Issue by droshev** [SAP/fundamental#1337]
**Describe the bug**
there are a number of locations in our components where fiori fundamentals hard codes `<h1>`. for Accessibility (A11y) purposes, these *must* be configurable. Additionally, they should probably provide H3 as default, and probably should *not* provide H1 as a option (any page should only have a single h1)

> Is this issue related to a specific component?

> What version of the Fiori Fundamentals are you using?

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

> What front-end framework are you implementing in, e.g., Angular, React, etc.?

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by jbadan** [comment]
Looks like this is a copy of https://github.com/SAP/fundamental/issues/1178 --- currently in progress :)

**Comment by jbadan** [comment]
fixed by https://github.com/SAP/fundamental/pull/1367
